.github/renovate.json

```diff
@@ -201,11 +201,7 @@
       "groupName": "ciprod",      
       "ignoreUnstable": false,
       "reviewers": [
-        "ganga1980",
-        "suyadav1",
-        "wanlonghenry",
-        "zanejohnson-azure",
-        "joby-thomas"
+        "team:containerinsights"
       ],
       "assignees": [
         "ganga1980",
```

Comment:
should we do the same for assignees?

Reply:
assignee doesn't support team unfortunately 😂

Reply:
don't know why

Reply:
gotcha, sad